src/app/private/[uid]/user/layout.tsx

```diff
@@ -1,5 +1,6 @@
 "use client";
 
+import Sidebar, { ISideBar } from "@components/Sidebar";
```

Comment:
I don't think this import statement is necessary, since it's not used.

Reply:
I think the import is used though, note lines 14, 38 in the file.